seqr/management/tests/check_bam_cram_paths_tests.py

```diff
@@ -3,32 +3,43 @@
 from django.core.management import call_command
 from django.test import TestCase
 
+from seqr.models import IgvSample
 
+
+@mock.patch('hail.hadoop_is_file', lambda path: False)
 class CheckBamCramPathsTest(TestCase):
     fixtures = ['users', '1kg_project']
 
+    def setUp(self):
+        existing_sample = IgvSample.objects.first()
+        IgvSample.objects.create(
+            individual=existing_sample.individual,
+            sample_type=IgvSample.SAMPLE_TYPE_GCNV,
+            file_path='gs://missing-bucket/missing_file',
+        )
+
     @mock.patch('seqr.management.commands.check_bam_cram_paths.logger')
-    @mock.patch('seqr.views.utils.dataset_utils.validate_alignment_dataset_path')
-    def test_command(self, mock_validate_path, mock_logger):
+    def test_command_with_project(self, mock_logger):
         call_command('check_bam_cram_paths', '1kg project n\u00e5me with uni\u00e7\u00f8de')
 
-        calls = [
-            mock.call('---- DONE ----'),
-            mock.call('Checked 1 samples'),
-            mock.call('0 failed samples: '),
-        ]
-        mock_logger.info.assert_has_calls(calls)
-        mock_validate_path.assert_called_with("/readviz/NA19675.cram")
+        self._check_results(mock_logger)
 
-        # Test exception
-        mock_validate_path.side_effect = Exception('Error accessing "/readviz/NA19675.cram"')
-        call_command('check_bam_cram_paths', '1kg project n\u00e5me with uni\u00e7\u00f8de')
+    @mock.patch('seqr.management.commands.check_bam_cram_paths.logger')
+    def test_command(self, mock_logger):
+        # run on just the 1kg project
+        call_command('check_bam_cram_paths')
+
+        self._check_results(mock_logger)
+
+    def _check_results(self, mock_logger):
+        self.assertEqual(IgvSample.objects.filter(file_path='').count(), 1)
+        self.assertEqual(IgvSample.objects.count(), 2)
 
         calls = [
-            mock.call('Error at /readviz/NA19675.cram (Individual: NA19675_1): Error accessing "/readviz/NA19675.cram" '),
+            mock.call('Individual: NA19675_1  file not found: gs://missing-bucket/missing_file'),
             mock.call('---- DONE ----'),
             mock.call('Checked 1 samples'),
-            mock.call('1 failed samples: NA19675_1'),
+            mock.call('1 files not found:'),
+            mock.call('   1 in 1kg project nåme with uniçøde'),
         ]
         mock_logger.info.assert_has_calls(calls)
-        mock_validate_path.assert_called_with("/readviz/NA19675.cram")
```

Comment:
Its important to test that the mock hail check is being called the way we expect